packages/nitro-protocol/test/contracts/ForceMove/forceMove.test.ts

```diff
@@ -190,39 +190,30 @@ describe('forceMove', () => {
       } else {
         const receipt = await (await tx).wait();
         await writeGasConsumption('./forceMove.gas.md', description, receipt.gasUsed);
+
+        // Catch ChallengeRegistered event
         const event = receipt.events.pop();
 
-        // Catch ForceMove event
-        const {
-          channelId: eventChannelId,
-          turnNumRecord: eventTurnNumRecord,
-          finalizesAt: eventFinalizesAt,
-          challenger: eventChallenger,
-          isFinal: eventIsFinal,
-          fixedPart: eventFixedPart,
-          variableParts: eventVariableParts,
-        } = event.args;
+        // Check this information is enough to respond or checkpoint
+        // (fixed part is assumed known)
+
+        const formatVariablePartForEthers = (variablePart: VariablePart) => [
+          variablePart.outcome,
+          variablePart.appData,
+        ];
+        const formatSignatureForEthers = (sig: Signature) => [sig.v, sig.r, sig.s];
```

Comment:
This is quite a bit neater and transparent IMO. Likely to change when we upgrade ethers.